### ISSUE TITLE
Object intensity measurements: Add background subtraction

### ISSUE THREAD

**Issue by tischi** [NEUBIAS/training-resources#281]
@manerotoni @maulakhan 

I think we should really add this somewhere:

`Mean_Ratio = ( meanA - BG ) / (meanB - BG)`
`Sum_Ratio = ( sumA - nPixelsA * BG ) / (sumB - nPixelsB * BG)`

Where should we add it? I think we decided [not to add it here](https://neubias.github.io/training-resources/measure_intensities/index.html) and just use already BG-corrected images, but maybe that sends the wrong message.

We could rename: [Global Background Correction](https://neubias.github.io/training-resources/global_background_correction/index.html) to `Background Measurement and Correction` and then this module would be a prerequisite for [Object intensity measurements](https://neubias.github.io/training-resources/measure_intensities/index.html).

What do you think?

**Comment by manerotoni** [comment]
Yes I think it belongs to Background assessment and correction. 
Would it make sense to reiterate the principle in the exercise with an image that is not background corrected. The students should spot the problem?

**Comment by tischi** [comment]
> Would it make sense to reiterate the principle in the exercise with an image that is not background corrected. 

To keep it simple I would then _only_ do an image without background correction, because that's what you typically have.

**Comment by tischi** [comment]
I'll just try to this now to object intensity measurements (because I need to teach this on Tuesday). We can then refactor those modules at some other time.

**Comment by tischi** [comment]
Concept map looks like this now:

<img width="867" alt="image" src="https://user-images.githubusercontent.com/2157566/156918289-0e31dcce-8d81-4643-9a95-9c4297a2091d.png">

The module does become more complex, but I think that's life...
